my/index.md

```diff
@@ -14,8 +14,7 @@ _Maklumbalas dan penyertaan anda amat dialu-alukan!_
 
 "Elixir ialah satu bahasa aturcara kefungsian(functional language) yang dinamik direka untuk membina aplikasi yang mudah untuk dinaiktaraf(scalable) dan mudah untuk diselenggara." - [elixir-lang.org](http://elixir-lang.org/)
 
-Elixir leverages the battle tested ErlangVM to build distributed and fault-tolerant systems with low-latency out of the box.
-Elixir meng-leverage ErlangVM untuk membina sistem yang bersifat mudah-agih(distributed) dan berdayatahan(fault-tolerant) serta mempunyai redah-kependaman(low-latency) yang disiap-pasang.
+Elixir menuilkan kelebihan ErlangVM untuk membina sistem yang bersifat mudah-agih(distributed) dan berdayatahan(fault-tolerant) serta mempunyai redah-kependaman(low-latency) yang disiap-pasang.
```

Comment:
Some of the other translators have started working on a Glossary for terms like these, if you want to check it out: https://github.com/doomspork/elixir-school/blob/master/GLOSSARY.md